akka-actor-typed-tests/src/test/scala/akka/actor/typed/AskSpec.scala

```diff
@@ -26,43 +29,38 @@ class AskSpec extends TypedSpec with ScalaFutures {
   val behavior: Behavior[Msg] = immutable[Msg] {
     case (_, foo: Foo) ⇒
       foo.replyTo ! "foo"
-      same
+      Actor.same
     case (_, Stop(r)) ⇒
       r ! ()
-      stopped
+      Actor.stopped
   }
 
   "Ask pattern" must {
     "must fail the future if the actor is already terminated" in {
-      val fut = for {
-        ref ← system ? TypedSpec.Create(behavior, "test1")
-        _ ← ref ? Stop
-        answer ← ref.?(Foo("bar"))(Timeout(1.second), implicitly)
-      } yield answer
-      fut.recover { case _: AskTimeoutException ⇒ "" }.futureValue should ===("")
+      val ref = spawn(behavior)
+      (ref ? Stop).futureValue
```

Comment:
We seem to use `.futureValue` and `Await`, should we pick one? `Await` is a bit verbose (requiring the `atMost` parameter), `futureValue` is shorter but not a great name IMHO.

Perhaps we should introduce convenience methods like `Future.get`, `Future.result`, `Future.await`?

Reply:
agreed futureValue isn't the best name but it is standard in Scalatest so maybe best to just stick with it and make the test consistent?

Reply:
makes sense 👍